pkg/cache/informer_cache.go

```diff
@@ -120,7 +120,8 @@ func (ip *informerCache) GetInformerForKind(gvk schema.GroupVersionKind) (Inform
 	if err != nil {
 		return nil, err
 	}
-	_, i, err := ip.InformersMap.Get(gvk, obj)
+
+	_, i, err := ip.InformersMap.Get(context.TODO(), gvk, obj)
```

Comment:
can you add a TODO here explaining what we'd need to do to plumb this through?

Reply:
My best attempt at an explanation. WDYT?

Reply:
sure, looks good.  Do you mind putting together a follow-up to do that?